docs/xtrabackup_bin/restoring_individual_tables.md

```diff
@@ -12,59 +12,47 @@ a int(11) DEFAULT NULL
 ```{.bash data-prompt="$"}
 $ find /data/backups/mysql/ -name export_test.*
 /data/backups/mysql/test/export_test.ibd
 ```
 
-when you prepare the backup, add the –export option to the
+when you prepare the backup, add the --export option to the
 command. Here is an example:
 
 ```{.bash data-prompt="$"}
 $ xtrabackup --prepare --export --target-dir=/data/backups/mysql/
 ```
 
-!!! note
-   
-    If you restore an encrypted InnoDB tablespace table, add the  keyring file:
+If you restore an encrypted InnoDB tablespace table, add the keyring file:
 
-    ```{.bash data-prompt="$"}
-    $ xtrabackup --prepare --export --target-dir=/tmp/table \
-    --keyring-file-data=/var/lib/mysql-keyring/keyring
-    ```
+```{.bash data-prompt="$"}
+$ xtrabackup --prepare --export --target-dir=/tmp/table \
+--keyring-file-data=/var/lib/mysql-keyring/keyring
+```
 
-Now you should see an .exp file in the target directory:
+Now you should see the `export_test.ibd` and `export_test.cfg` files in the target directory:
 
 ```{.bash data-prompt="$"}
 $ find /data/backups/mysql/ -name export_test.*
-/data/backups/mysql/test/export_test.exp
 /data/backups/mysql/test/export_test.ibd
 /data/backups/mysql/test/export_test.cfg
 ```
 
-These three files are the only files required to import the table into a server running
-*Percona Server for MySQL* with *XtraDB* or *MySQL* 8.0. In case the server uses InnoDB,
+These two files are the only files required to import the table into a server running
+Percona Server for MySQL with XtraDB or MySQL 8.0. In case the server uses InnoDB,
 Tablespace Encryption adds an additional .cfp file which contains the transfer key and an encrypted tablespace key.
 
-!!! note
-   
-    The .cfg metadata file contains an *InnoDB* dictionary dump in a special format. This format is different from the .exp one which is
-    used in *XtraDB* for the same purpose. A .cfg\` file is not required to import a tablespace to *MySQL* 8.0 or Percona
-    Server for MySQL 8.0.
-
-    A tablespace is imported successfully even if the table is from
-    another server, but *InnoDB* performs a schema validation if the corresponding .cfg file is located in the same directory.
-
 ## Import the table
 
-On the destination server running *Percona Server for MySQL* with *XtraDB* and
-`innodb_import_table_from_xtrabackup` option enabled, or *MySQL* 8.0, create a table with the same structure, and then perform the following steps:
+On the destination server running Percona Server for MySQL with XtraDB and
+`innodb_import_table_from_xtrabackup` option enabled, or MySQL 8.0, create a table with the same structure, and then perform the following steps:
 
-1. Run the `ALTER TABLE test.export_test DISCARD TABLESPACE;` command. If you see the following error, enable innodb_file_per_table and create the table again.
+1. Run the `ALTER TABLE test.export_test DISCARD TABLESPACE;` command. If you see the following error, enable `innodb_file_per_table` and create the table again.
```

Comment:
I don't see the error description?

Reply:
error: ERROR 1809 (HY000): Table 'test/t1' in system tablespace

How to enable:
`set global innodb_file_per_table=ON;`

Reply:
Updated